cacerts/testdata/SecureTrust_CA-corrupt.pem

```diff
@@ -0,0 +1,19 @@
+NVBAMTDlNlY3VyZVRydXN0IENBMB4XDTA2MTEwNzE5MzExOFoXDTI5MTIz
```

Comment:
There was a file `multiple-certs.pem` referenced from the test above, but I don't see that being generated or included in the testdata. Am I missing something?

Reply:
This should be in the `testdata` dir with the other test files?

Reply:
Lol, that's my fault. I assumed that it was a new file and didn't see it in the commit. All good.